vips/vip-289/bsctestnet.ts

```diff
@@ -0,0 +1,244 @@
+    amount: parseUnits("37.686205910410599126", "18"),
+  },
+  {
+    address: "0xbcC32d0729043259594C36Ac53c3D2f94CD61860",
+    amount: parseUnits("29.576315531093403723", "18"),
+  },
+  {
+    address: "0x7df11563c6b6b8027aE619FD9644A647dED5893B",
+    amount: parseUnits("15.333333466222650063", "18"),
+  },
+  {
+    address: "0xC8E2d418dAE56DcC9fa05A1c67AfeD876F04e553",
+    amount: parseUnits("10", "18"),
+  },
+  {
+    address: "0xD4a9F68DEbed2F8D7cE0115d415cc992AB0BFbcc",
+    amount: parseUnits("1", "18"),
+  },
+  {
+    address: "0x9CeF4778267400D921271f0BC9434C3f9a73d826",
+    amount: parseUnits("1", "18"),
+  },
+  {
+    address: "0x25c7c7D6Bf710949fD7f03364E9BA19a1b3c10E3",
+    amount: parseUnits("0.313794089589400874", "18"),
+  },
+];
+
+let MINT_TOTAL = parseUnits("0", "18");
+ankrBNB_TOKEN_HOLDERS.forEach(holder => {
+  MINT_TOTAL = MINT_TOTAL.add(holder.amount);
+});
+
+const TRANSFER_COMMANDS = ankrBNB_TOKEN_HOLDERS.map(holder => {
+  return {
+    target: ankrBNB,
+    signature: "transfer(address,uint256)",
+    params: [holder.address, holder.amount],
+  };
+});
+
+const vip289 = () => {
+  const meta = {
+    version: "v2",
+    title: "VIP-283",
+    description: ``,
+    forDescription: "Execute this proposal",
+    againstDescription: "Do not execute this proposal",
+    abstainDescription: "Indifferent to execution",
+  };
+
+  return makeProposal(
+    [
+      {
+        target: RESILIENT_ORACLE,
+        signature: "setTokenConfig((address,address[3],bool[3]))",
+        params: [
+          [
+            BNBx,
+            [BNBxOracle, "0x0000000000000000000000000000000000000000", "0x0000000000000000000000000000000000000000"],
+            [true, false, false],
+          ],
+        ],
+      },
+      {
+        target: RESILIENT_ORACLE,
+        signature: "setTokenConfig((address,address[3],bool[3]))",
+        params: [
+          [
+            SlisBNB,
+            [SlisBNBOracle, "0x0000000000000000000000000000000000000000", "0x0000000000000000000000000000000000000000"],
+            [true, false, false],
+          ],
+        ],
+      },
+      {
+        target: RESILIENT_ORACLE,
+        signature: "setTokenConfig((address,address[3],bool[3]))",
+        params: [
+          [
+            StkBNB,
+            [StkBNBOracle, "0x0000000000000000000000000000000000000000", "0x0000000000000000000000000000000000000000"],
+            [true, false, false],
+          ],
+        ],
+      },
+      {
+        target: WBETH,
+        signature: "setExchangeRate(uint256)",
+        params: [WBETH_EXCHANGE_RATE],
+      },
+      {
+        target: RESILIENT_ORACLE,
+        signature: "setTokenConfig((address,address[3],bool[3]))",
+        params: [
+          [
+            WBETH,
+            [WBETHOracle, "0x0000000000000000000000000000000000000000", "0x0000000000000000000000000000000000000000"],
+            [true, false, false],
+          ],
+        ],
+      },
```

Comment:
We won't change the oracle for WBETH on mainnet, so I wouldn't change it on testnet either